src/js/components/search/filters/programSource/ProgramSourceSection.jsx

```diff
@@ -196,7 +183,7 @@ export default class ProgramSourceSection extends React.Component {
             </React.Fragment>
         );
 
-        const tab2Title = isDev ? 'TAS Components' : 'Federal Account';
+        const tab2Title = 'TAS Components';
```

Comment:
Looks like this doesn't need to be a variable anymore (could be hard-coded on line 214)